graphql-dgs-graphiql-autoconfigure/build.gradle.kts

```diff
@@ -16,9 +16,23 @@
 
 plugins {
     kotlin("jvm")
+	kotlin("kapt")
 }
 
 dependencies {
     implementation("org.springframework.boot:spring-boot-autoconfigure")
     implementation("org.springframework:spring-webmvc")
-}
\ No newline at end of file
+
+	kapt("org.springframework.boot:spring-boot-configuration-processor:${Versions.SPRING_BOOT_VERSION}")
+
+	testImplementation("org.springframework.boot:spring-boot-starter-web")
+}
+
+kapt {
```

Comment:
This should not be necessary and is probably a bug in kapt or the boot config processor - trust me, it does not include the metadata w/o this - it robbed me of 1-2 hrs of time. It works w/ this though. We will probably want to factor this out for easier re-use when we start adding config props.